easse/bleu.py

```diff
@@ -0,0 +1,54 @@
+"""
+Implements the evaluation metrics based on BLEU score
+"""
+
+import numpy as np
+from typing import List
+
+import sacrebleu
+
+import easse.utils.preprocessing as utils_prep
+
+
+def corpus_bleu(sys_sents: List[str],
+                refs_sents: List[List[str]],
+                smooth_method: str = 'exp',
+                smooth_value: float = None,
+                force: bool = True,
+                lowercase: bool = False,
+                tokenizer: str = '13a',
+                use_effective_order: bool = False):
+
+    sys_sents = [utils_prep.normalize(sent, lowercase, tokenizer) for sent in sys_sents]
+    refs_sents = [[utils_prep.normalize(sent, lowercase, tokenizer) for sent in ref_sents]
+                  for ref_sents in refs_sents]
+
+    return sacrebleu.corpus_bleu(sys_sents, refs_sents, smooth_method, smooth_value, force,
+                                 lowercase=False, tokenize='none', use_effective_order=use_effective_order).score
+
+
+def sentence_bleu(sys_sent: str,
+                  ref_sents: List[str],
+                  smooth_method: str = 'floor',
+                  smooth_value: float = None,
+                  lowercase: bool = False,
+                  tokenizer: str = '13a',
+                  use_effective_order: bool = True):
+
+    return corpus_bleu([sys_sent], [[ref] for ref in ref_sents], smooth_method, smooth_value, force=True,
+                       lowercase=lowercase, tokenizer=tokenizer, use_effective_order=use_effective_order)
+
+
+def corpus_macro_sentence_bleu(sys_sents: List[str],
```

Comment:
I know it's nitpicking and I already made the comment somewhere else:
Don't you think something like `corpus_averaged_sentence_bleu` would be more readable?
Of course we would probably also need to rename other functions that use the "macro" term.
I feel that "macro" is a bit weird / not explicit...

Reply:
I have no problem with that change here.